### ISSUE TITLE
`DeprecationWarning` when using `random.shuffle` with `jax` backend

### ISSUE THREAD

**Issue by KhawajaAbaid** [keras-team/keras#19271]
Hi,
when we use the `random.shuffle` method with `jax` backend in Keras, it raises the following warning,

```
DeprecationWarning: jax.random.shuffle is deprecated. Use jax.random.permutation with independent=True.
```

I have a PR draft ready that fixes this warning by replacing the following call in the existing `shuffle` function 
```
    return jax.random.shuffle(seed, x, axis)
```
with the call to permutation instead as suggested by the warning.
```
    return jax.random.permutation(seed, x, axis, independent=True)
```

Please let me know if I should go ahead and create this PR.

Thanks.

**Comment by KhawajaAbaid** [comment]
Okay, i understand that it was a one line update and not a big deal, but it's still rude to yoink something i proposed and create your own PR when I've explicitly stated I have a PR draft ready and I'm asking for your approval to create it.

Also if this is what one gets for following the contribution instructions to first create an issue and then open a PR, then it's really sad.

Not making this comment to cause any drama, but doing so merely to highlight so people in future don't have their work robbed.

Have a great day!

**Comment by sachinprasadhs** [comment]
Apologies, Didn't notice the last part. 
Was focusing on validating the other dependencies with the fix.

Going forward, you can directly create PR with the description without needing to create an issue.
Thanks for your contribution.
